### PR TITLE
BY api for customer transaction table

### PR DESCRIPTION
**Pull request by Beiancaa**

### CODE REVIEW

File: app/controllers/api/v1/customers_controller.rb [hunk 1]
```diff
@@ -15,8 +15,12 @@ def deactive
   # end
 
   def customer_token_transaction
-    @transactions = TokenTransaction.where(customer_id: current_customer.id).order('created_at DESC, id DESC')
-    @debit_count = @transactions.where(tran_process: 'Withdrawal').count
-    @credit_count = @transactions.count - @debit_count
+    begin
+      transactions = TokenTransaction.where(customer_id: current_customer.id).paginate(page: params[:page], per_page: params[:per_page] ||= 10).order('created_at DESC, id DESC')
```
Comment (ssharba) [inline app/controllers/api/v1/customers_controller.rb]: don't use current_customer, get it as params please.

File: config/routes.rb [hunk 2]
```diff
@@ -6,6 +6,9 @@
       resources :survey_forms, only: [:show] do
       end
       resources :customers, only: [:create] do
+        collection do
```
Comment (ssharba) [inline config/routes.rb]: make it as member so you will get customer_id in the url

**Comment by ssharba** [comment]
@Beiancaa conflicts

**Comment by Beiancaa** [comment]
@ssharba  fixed

**Pull request merged**